utils/filter.c

```diff
@@ -808,6 +822,10 @@ static const struct trigger_action_parser actions[] = {
 		"auto-args",
 		parse_auto_args_action,
 	},
+	{
+		"clear",
+		parse_clear_action,
+	},
```

Comment:
Can you please update the documentation?

Reply:
I added a few lines in `uftrace-live.md` and `uftrace-record.md` to explain that the `@clear` suffix exists. I plan to add a reference to the agent section when I create it, where I will show examples. Is it OK for now? Or do you want to have examples in the doc in this PR?

Reply:
You can do it in a separate PR.  Thanks!